logger/logger.go

```diff
@@ -0,0 +1,62 @@
+package logger
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var logger *logrus.Logger
+
+// Fields provides a type, used for global callers in image-spec, with accordance to logrus.Fields
+type Fields logrus.Fields
+
+type stderrHook struct{}
+
+// Levels provide the method for logrus.Hook interface
+func (h *stderrHook) Levels() []logrus.Level {
+	return []logrus.Level{
+		logrus.PanicLevel,
+		logrus.FatalLevel,
+		logrus.ErrorLevel,
+		logrus.WarnLevel,
+	}
+}
+
+// Fire provide the method for logrus.Hook interface
+func (h *stderrHook) Fire(entry *logrus.Entry) error {
+	logOfEntry := reflect.ValueOf(*entry.Logger).Interface().(logrus.Logger)
+	entry.Logger = &logOfEntry
+	entry.Logger.Out = os.Stderr
+	return nil
+}
+
+// NewLogger returns the logger.
+func NewLogger() {
+	logger = logrus.StandardLogger()
```

Comment:
This isn't very good practice, nor is it threadsafe. Typically, `NewXXX` functions return a new thing. Side-effecting an unprotected global is really not good practice.